### ISSUE TITLE
temperature annealing

### ISSUE THREAD

**Issue by wbchao1997** [prstrive/CondConv-tensorflow#2]
The temperature reduces with the training epochs can further improves the accuracy according to the paper, do you know how to implement this? I didn't figure out a proper way using tensorflow.

**Comment by prstrive** [comment]
The temperature is an attribute of class Routing. Therefore,  its value can be modified in each epoch.  What you need to do is to use the Tensorflow API to find the Routing object of the model, and then directly assign a new value to the temperature of the Routing object.
